1_uprava.js

```diff
@@ -0,0 +1,8 @@
+const countVowels = (text) => {
```

Comment:
Toto je len ES6 sposob zapisu funkcie.

Namiesto
`function nazovA(parameterA) {}`
Napises
`const nazovA = (parameterA) => {}`

Reply:
Taktiez som premenoval nazov funkcie z `countV` na `countVowels`. Skracovanie nazvu funkcii / premennych je fajn, pokym ale dokazes ihned sposnaz co data funckia robi, zakladny princip clean codu.

Pri nazve `countV` mi nazov skor hovori o pocitani pismen `V` ako o tom ze to ma nieco spolocne so samohlaskami